doc/source/reference/ufuncs.rst

```diff
@@ -651,9 +652,11 @@ single operation.
     isfinite
     isinf
     isnan
+    fabs
```

Comment:
Should probably add this to the arithmetic functions as well. Note that `fmod` is in both places.